### DISCUSSION TITLE
Can I fill in the local path in the RESOURCE_PACK variable? For example: /data/RESOURCE_PACK/RESOURCE_PACK_1.zip

### DISCUSSION THREAD

**Discussion by djdjfjfkeocj** [itzg/docker-minecraft-server#2856] [Q&A]
Can I fill in the local path in the RESOURCE_PACK variable?
For example: /data/RESOURCE_PACK/RESOURCE_PACK_1.zip

**Comment by itzg** [comment]
No, the Minecraft server needs it to be a URL that the client can access also.

**Reply by djdjfjfkeocj** [reply]
all right,thanks

**Reply by djdjfjfkeocj** [reply]
What if you want multiple resource packs

**Reply by itzg** [reply]
Please consult the Minecraft docs. The container startup only passes through the given value.
